### ISSUE TITLE
Support multi-root workspaces

### ISSUE THREAD

**Issue by doppelganger9** [alexandrudima/vscode-lcov#18]
Right now, the extension seems to look for the `lcov.info` file in the first folder within a [multi-root workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces).

When I open my folder alone it finds the `lcov.info` correctly.

**Comment by jedwards1211** [comment]
Yeah when I configure it to use `./coverage/lcov.info`, I wish it would look for this relative to every Git root folder (I know not all VSCode projects will use Git, but this would benefit the majority of projects)
